mup_scripts/mutransfer_lr/plot.ipynb

```diff
@@ -0,0 +1,154 @@
+    "\n",
+    "    def __init__(self, cmap_name, start_val, stop_val):\n",
+    "        self.cmap_name = cmap_name\n",
+    "        self.cmap = plt.get_cmap(cmap_name)\n",
+    "        self.norm = mpl.colors.Normalize(vmin=start_val, vmax=stop_val)\n",
+    "        self.scalarMap = cm.ScalarMappable(norm=self.norm, cmap=self.cmap)\n",
+    "\n",
+    "    def get_rgb(self, val):\n",
+    "        return self.scalarMap.to_rgba(val)\n",
+    "\n",
+    "\n",
+    "color_helper = MplColorHelper('viridis', 0, len(widths)-1)\n",
+    "n_cols = 2#len(parameterizations)\n",
+    "n_rows = 1\n",
+    "fig, axes = plt.subplots(n_rows, n_cols, figsize=(8*n_cols, 6*n_rows))\n",
+    "# axes = axes.reshape((n_rows, n_cols))\n",
+    "\n",
+    "for parameterization_idx, (parameterization, parameterization_str) in enumerate(parameterizations):\n",
+    "    ax = axes[parameterization_idx]\n",
+    "    optimal_lrs = []\n",
+    "    optimal_losses = []\n",
+    "    for width_idx, width in enumerate(widths):\n",
+    "        mean_losses = []\n",
+    "        sem_losses = []\n",
+    "        lrs_to_plot = []\n",
+    "        for lr in lrs:\n",
+    "            losses = []\n",
+    "            for seed in seeds:\n",
+    "                job_name = f'width{width}_depth2_seed{seed}_lr{lr:.20f}'.rstrip('0')\n",
+    "                csv_path = os.path.join(parameterization, 'out', job_name, 'log.csv')\n",
+    "                if os.path.exists(csv_path):\n",
+    "                    ckpt_df = pd.read_csv(csv_path)\n",
+    "                    losses.append(ckpt_df['train/loss'].mean())\n",
+    "                    #print(losses)\n",
+    "                #else:\n",
+    "                    #print(f'Missing {csv_path}')\n",
+    "            if len(losses):\n",
+    "                mean_losses.append(np.mean(losses))\n",
+    "                sem_losses.append(np.std(losses, ddof=1) / np.sqrt(len(losses)))\n",
+    "                lrs_to_plot.append(lr)\n",
+    "        \n",
+    "        mean_losses = np.array(mean_losses)\n",
+    "        sem_losses = np.array(sem_losses)\n",
+    "        ax.plot(lrs_to_plot, mean_losses, label=width, marker='o', color=color_helper.get_rgb(width_idx))\n",
+    "        ax.fill_between(lrs_to_plot, mean_losses-sem_losses, mean_losses+sem_losses, color=color_helper.get_rgb(width_idx), alpha=0.5)\n",
+    "            \n",
+    "        if len(mean_losses):\n",
+    "            optimum_idx = np.argmin(mean_losses)\n",
+    "            optimal_lrs.append(lrs_to_plot[optimum_idx])\n",
+    "            optimal_losses.append(mean_losses[optimum_idx])\n",
+    "        \n",
+    "    ax.plot(optimal_lrs, optimal_losses, color='red', linestyle='none', marker='o')\n",
+    "    ax.set_xscale('log', base=2)\n",
+    "    ax.set_xlabel('Learning rate')\n",
+    "    ax.set_title(parameterization_str)\n",
+    "    ax.set_ylim(7, 9)\n",
+    "\n",
+    "axes[0].legend(title='Width')\n",
```

Comment:
Maybe `axes[1].legend(title='Width')` will look better since the muP curves leave some negative space

Reply:
Looks better staying as `axes[0].legend(title='Width')` with the new figures